synfig-studio/src/gui/canvasview.cpp

```diff
@@ -705,18 +707,12 @@ CanvasView::~CanvasView()
 	SYNFIG_EXCEPTION_GUARD_END()
 }
 
-void CanvasView::save_all()
-{
-	std::list<etl::handle<studio::Instance> >::iterator iter;
-	for(iter=App::instance_list.begin();iter!=App::instance_list.end();iter++)
-		(*iter)->save();
-}
-
 void CanvasView::activate()
 {
 	activation_index_.activate();
 	get_smach().process_event(EVENT_REFRESH_TOOL_OPTIONS);
 	App::ui_manager()->insert_action_group(action_group);
+	App::main_window->insert_action_group("doc", action_group_);
```

Comment:
It is inserted here, @ice0 .

Reply:
And it works on Gnome.

Reply:
Okay, I will ask the GTK team about it then.